content/docs/user-guide/visualization/animation/animation-editor/ragdoll/ragdoll-physics-setup.md

```diff
@@ -0,0 +1,145 @@
+
+    ![Use the Ragdoll joints and colliders filter to show only joints in the ragdoll in the Animation Editor](/images/user-guide/actor-animation/ragdoll-skeleton-outliner-ragdoll-joints-colliders-filter.png)
+
+    The **Skeleton Outliner** displays icons to indicate that:  
+    ![Icon showing that a joint is part of the ragdoll](/images/user-guide/actor-animation/ragdoll-skeleton-outliner-joint-icons-joint-limit.svg) A joint is part of the ragdoll  
+    ![Icon showing that a joint holds ragdoll colliders](/images/user-guide/actor-animation/ragdoll-skeleton-outliner-joint-icons-collider.svg) A joint holds ragdoll colliders  
+    ![Icon showing that a joint holds hit detection colliders](/images/user-guide/actor-animation/ragdoll-skeleton-outliner-joint-icons-hit-detection.svg) A joint holds hit detection colliders  
+    ![Icon showing that a joint holds cloth colliders](/images/user-guide/actor-animation/ragdoll-skeleton-outliner-joint-icons-cloth.svg) A joint holds cloth colliders  
+
+    ![Icons in the Skeleton Outliner show how joints are related to the ragdoll, ragdoll colliders, and hit detection colliders](/images/user-guide/actor-animation/ragdoll-skeleton-outliner-joint-icons.png)
+
+1. In the **Atom Render Window**, use the render options dropdown, represented by the ![Use the render options in the Atom Render Window to show or hide ragdoll colliders and joint limits and hit detection colliders](/images/user-guide/actor-animation/ragdoll-skeleton-render-options.svg) icon, to show or hide the ragdoll colliders (rendered in orange), ragdoll joint limits, hit detection colliders (rendered in blue), and cloth colliders (rendered in purple).
+
+    {{< note >}}
+If your ragdoll colliders, hit detection colliders, or cloth colliders are the same size, you may need to hide the colliders that you are not working on.
+{{< /note >}}
+
+1. On the **Ragdoll** tab, you can view and modify the ragdoll properties for the selected joint. For example, the rigid body mass, sleeping threshold, and colliders.
+
+![View and modify ragdoll properties for a selected joint on the Ragdoll tab in the Animation Editor](/images/user-guide/actor-animation/ragdoll-skeleton-ragdoll-tab-properties.png)
+
+## Step 2: Set up Ragdoll Colliders 
+
+**Adding and removing ragdoll colliders**
+
+Each joint in the ragdoll can have 0, 1 or multiple ragdoll colliders, which affect how the ragdoll collides with other physical objects. Box, sphere and capsule geometries are supported. By default, when a joint is added to the ragdoll, it is automatically created with a 
+capsule collider which roughly approximates the shape of the parts of the actor mesh most heavily influenced by that joint. If a different geometry would work better, you can delete the default collider and add another as described below.
```

Comment:
nit: No need to line break here, wrapping should be handled correctly when rendered on the website

Reply:
Not sure how that happened, fixed.